src/Microsoft.Data.SqlClient/ref/Microsoft.Data.SqlClient.Batch.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Data.SqlClient
+{
+    /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlBatch.xml' path='docs/members[@name="SqlBatch"]/SqlBatch/*'/>
+    public class SqlBatch : System.Data.Common.DbBatch
+    {
+        /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlBatch.xml' path='docs/members[@name="SqlBatch"]/ctor1/*'/>
+        public SqlBatch() { throw null; }
+        /// <include file='../../../doc/snippets/Microsoft.Data.SqlClient/SqlBatch.xml' path='docs/members[@name="SqlBatch"]/ctor2/*'/>
+        public SqlBatch(Microsoft.Data.SqlClient.SqlConnection connection, Microsoft.Data.SqlClient.SqlTransaction transaction = null) { throw null; }
+        /// <inheritdoc />
```

Comment:
I wish I had known about this element a long time ago... 😮 Are there really that many places where our batching implementation will require different docs than the common API?

Reply:
I marked items which are overrides with inherit but anything which isn't an override requires it's own content.